Server/controllers/time-entry.controller.js

```diff
@@ -4,13 +4,124 @@
  *
  * @class timeEntriesController
  */
+const mongoose = require('mongoose');
+const timeEntryModel = require('../models/time-entry.model');
+const dataLayer = require('../datalayer/time-entry.datalayer')
+const moment = require('moment');
+
+
 class timeEntriesController {
 
-    static getTimeEntriesByUserId (req, res) {}
+    //  GET 
+
+    static async getTimeEntriesByUserId(req, res) {
+        let timeEntries;
+
+        const obj = {
+            "userId": req.query.user,
+            "limit": req.query.limit
+        }
+
+        try {
+            timeEntries = await dataLayer.getTimeEntriesByUserId(obj);
+        } catch (e) {
+            return res
+                .status(400)
```

Comment:
status 400 ?

Reply:
For now- Bad request is default
As dicussed, error handling will be done properly in future